<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QDialog" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="BitRate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>95</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>9</number>
   </property>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Parity">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>53</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>53</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>98</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Bitrate (bps)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ByteSize">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>7 bits</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 bits</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>308</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BtnOk">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnCancel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>53</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FlowCtr">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTS/CTS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XON/XOFF</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="StopBits">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>98</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>1 bit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 bits</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>130</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Byte Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>98</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Stop Bits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Port">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label8">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Flow Control</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
